app.py

```diff
@@ -22,51 +22,106 @@ class ReviewRequest(db.Model):
     pass
 
 
-reviewers = api.namespace("reviewers", description="Reviewer Management")
-
 #################
 # API Endpoints #
 #################
 
+def save_changes(data):
+    db.session.add(data)
+    db.session.commit()
+
+reviewers = api.namespace("reviewers", description="Reviewer Management")
+# TODO: Add public fields
+_reviewer =  api.model('reviewer', {})
 
 @reviewers.route("/")
 class ReviewerList(Resource):
+
+    @reviewers.doc('list of registered reviewers')
+    @reviewers.marshal_list_with(_reviewer, envelope='data')
     def get(self):
         """List of reviewers."""
-        pass
+        return Reviewer.query.all()
```

Comment:
What would you think about supporting URL request parameters to filter over the `Reviewers` returned? That would let a user hit the API for something like "give me a list of all Python reviewers who haven't done a review in the last two weeks" or the like.